packages/instantsearch.js/src/components/InternalHighlight/__tests__/InternalHighlight.test.tsx

```diff
@@ -1,210 +0,0 @@
-/**
```

Comment:
I see the argument that this falls under other tests, but doesn't this still assert that preact is properly used?

Reply:
Aren't the tests in `packages/instantsearch.js/src/helpers/components/__tests__/` enough? They're essentially the same and use `@testing-library/preact`'s `render`. The only difference is that we test the actual components, not this internal one.

Reply:
Ah yes, that makes sense. Good for me :)